helm/templates/ec2nodeclass.yaml

```
@@ -16,6 +16,12 @@ spec:
   amiFamily: AL2023
   amiSelectorTerms:
     - alias: al2023@v20251007
+  blockDeviceMappings:
+    - deviceName: /dev/xvda
+      ebs:
+        volumeSize: {{ .Values.volumeSize }} # 60Gi
```

Comment should be removed:

```suggestion
        volumeSize: {{ .Values.volumeSize }}
```